dbms/src/Flash/Mpp/MPPTask.cpp

```diff
@@ -578,6 +579,14 @@ void MPPTask::runImpl()
             runtime_statistics.blocks,
             runtime_statistics.bytes);
 
+        // Only report RU consumption to GAC for now.
+        // Will not affect the execution process of MPPTask,
+        // as the consult operation will only be performed after MPPTask is completed.
+        // NOTE: May throw if resource group is deleted.
```

Comment:
`May throw if resource group is deleted.`
catch and ignore the exception?

Reply:
done